### PR TITLE
feat: Add navbar, footer

### PR DESCRIPTION
**Pull request by Aditya062003**
Added Navbar and Footer

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,5 +1,18 @@
 function Footer() {
-  return <div>Footer</div>;
+  return (
+    <div>
+      <div className="bg-gray-800 text-center py-6 text-white">
+        {' '}
```
Comment (JaeAeich) [inline src/components/Footer/Footer.tsx:5]: remove this.

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+        Made with ♥ by{' '}
```
Comment (JaeAeich) [inline src/components/Footer/Footer.tsx:6]: same here

File: src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+    } catch (error) {}
+  }, []);
+  return (
+    <div className="flex justify-between items-center px-8 bg-[#CFC7E9] shadow-md py-4">
+      <div className="text-center">
+        <PixelText className="text-center">Pixels</PixelText>
+      </div>
+      <a href="https://github.com/JaeAeich/Pixels">
+        <div className="flex gap-2 items-center">
+          <div>⭐</div>
```
Comment (JaeAeich) [inline src/components/Navbar/Navbar.tsx:34]: use svg instead of emoji if possible

File: src/index.css [hunk 4]
```diff
@@ -6,6 +6,7 @@
   --color-saffron: #ff671f;
   --color-white: #ffffff;
   --color-green: #046a38;
+  --color-purple: #cfc7e9;
```
Comment (JaeAeich) [inline src/index.css:9]: Don't you think it should be --color-primary

File: src/components/Navbar/Navbar.tsx [hunk 5]
```diff
@@ -1,5 +1,42 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+const PixelText = styled.span`
```
Comment (JaeAeich) [inline src/components/Navbar/Navbar.tsx:4]: can we shift this to root or tailwind config so that entire app can use the same font?

**Review by JaeAeich — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.tsx (hunk 5 above)
Comment (Aditya062003) [reply src/components/Navbar/Navbar.tsx]: Resolved

File: src/components/Navbar/Navbar.tsx (hunk 3 above)
Comment (Aditya062003) [reply src/components/Navbar/Navbar.tsx]: Resolved

**Review by JaeAeich — APPROVED** [review]

File: src/index.css (hunk 4 above)
Comment (Aditya062003) [reply src/index.css]: Done

**Comment by JaeAeich** [comment]
@Aditya062003 Squash and merge :+1:

**Pull request merged**